node-tests/blueprints/controller-test-test.js

```diff
@@ -21,19 +21,25 @@ describe('Blueprint: controller-test', function() {
       return emberNew();
     });
 
-    it('controller-test foo', function() {
-      return emberGenerateDestroy(['controller-test', 'foo'], _file => {
-        expect(_file('tests/unit/controllers/foo-test.js')).to.equal(
-          fixture('controller-test/default.js')
-        );
+    describe('with ember-cli-qunit@4.1.0', function() {
+      beforeEach(function() {
+        generateFakePackageManifest('ember-cli-qunit', '4.1.0');
```

Comment:
Note: this is not really needed here. But AFAICT having `ember-cli-qunit` with a higher version in an addons `dependencies` would cause `ember-cli-blueprint-test-helpers` to run the blueprint in tests with that version, causing a different output than the tests expect. So we make the expected `ember-cli-qunit` explicit here, so the installed version does not matter, and we can use the same test as-is for the TS blueprints.

Reply:
This is exactly the scenario that I hit in https://github.com/emberjs/ember.js/pull/17126.